playground/src/components/entry-point/entry-point.js

```diff
@@ -1,12 +1,10 @@
 import React from 'react';
 import { Provider as StoreProvider } from 'react-redux';
-import Loadable from 'react-loadable';
 import {
   ApplicationShell,
   reduxStore,
   setupGlobalErrorListener,
   RouteCatchAll,
-  AsyncChunkLoader,
```

Comment:
I haven't removed this component yet because is still used by the applications. We might deprecate is at some point and then remove it.